tests/equilibria/test_flux_surfaces.py

```diff
@@ -40,6 +41,44 @@
 TEST_PATH = get_bluemira_path("equilibria/test_data", subfolder="tests")
 
 
+class TestFluxSurface:
+    def setup_class(self):
+        eq_name = "DN-DEMO_eqref.json"
+        filename = os.sep.join([TEST_PATH, eq_name])
+        self.eq = Equilibrium.from_eqdsk(filename)
+
+    def test_calculate_poloidal_angle(self):
+        # Flux surface in the SoL
+        x_start, z_start = 14, 0
+        x_loop, z_loop = find_flux_surface_through_point(
+            self.eq.x,
+            self.eq.z,
+            self.eq.psi(),
+            x_start,
+            z_start,
+            self.eq.psi(x_start, z_start),
+        )
+        fs = FluxSurface(Loop(x=x_loop, z=z_loop))
+        # Glancing angle
+        gamma = 5
+        # Poloidal angle
+        theta = fs.calculate_poloidal_angle(
+            self.eq,
+            10,
+            -7.5,
+            gamma,
+        )
+        assert theta > gamma
```

Comment:
```suggestion
        assert theta > gamma

```

Reply:
@sebkahn I don't understand this comment